pipeline/splicing/checkVarHGVS.py

```diff
@@ -0,0 +1,150 @@
+                  "varAlt":varAlt,
+                  "varType":varType}
+    return varParsed
+
+def getVarInfo(variant):
+    '''
+    Given a variant, returns a dictionary containing HGVS_cDNA, varType, varGenRef, varGenAlt, and depending on if varGene is BRCA1 or BRCA2 either the reverse complement or normal cDNA ref and alt alleles respectively
+    ''' 
+    varGenRef = variant["Ref"]
+    varGenAlt = variant["Alt"]
+    varcDNAHGVS = variant["pyhgvs_cDNA"]
+    varGene = variant["Gene_Symbol"]
+    varParse = parseVar(varcDNAHGVS)
+    varType = varParse["varType"]
+    varInfo = {"varGenRef":varGenRef,
+               "varGenAlt":varGenAlt,
+               "varcDNAHGVS":varcDNAHGVS,
+               "varType":varType,
+               "varGene":varGene}
+    if varGene == "BRCA2":
+        cDNARef = varParse["varRef"]
+        cDNAAlt = varParse["varAlt"]
+    elif varGene == "BRCA1":
+        # gets reverse complement because BRCA1 on minus strand
+        cDNARef = getRevComp(varParse["varRef"])
+        cDNAAlt = getRevComp(varParse["varAlt"])
+    else:
+        # varGene is not BRCA1 or BRCA2
+        cDNARef = varParse["varRef"]
+        cDNAAlt = varParse["varAlt"]
+    varInfo["cDNARef"] = cDNARef
+    varInfo["cDNAAlt"] = cDNAAlt
+    return varInfo
+    
+def checkVarHGVS(variant):
+    '''
+    Checks that ref and alt alleles for a variant are consistent with parsed HGVS_cDNA
+    If consistent, returns True
+    Otherwise returns a dictionary with 2 entries
+    "Consistent":False
+    and
+    "varData": a list of information about the variant that includes:
+        HGVS_cDNA, variant type, ref allele, alt allele, parsed cDNA ref allele, and parsed cDNA alt allele
+    '''
+    varInfo = getVarInfo(variant)
+    varType = varInfo["varType"]
+    varGenRef = varInfo["varGenRef"]
+    varGenAlt = varInfo["varGenAlt"]
+    cDNARef = varInfo["cDNARef"]
+    cDNAAlt = varInfo["cDNAAlt"]
+    if varType == ">" or varType == "delins":
+        if varGenRef == cDNARef and varGenAlt == cDNAAlt:
+            varEqual = True
+        else:
+            # varGenRef != cDNARef and varGenAlt != cDNAAlt
+            varEqual = False
+    elif varType == "del":
+        tempVarGenRef = varGenRef[1:]
+        if tempVarGenRef == cDNARef:
+            varEqual = True
+        else:
+            varEqual = False
+    elif varType == "dup":
+        tempVarGenAlt = varGenAlt[1:]
+        if tempVarGenAlt == cDNARef:
+            varEqual = True
+        else:
+            varEqual = False
+    else:
+        # varType == "ins":
+        tempVarGenAlt = varGenAlt[1:]
+        if tempVarGenAlt == cDNAAlt:
+            varEqual = True
+        else:
+            varEqual = False
+    if varEqual == True:
+        return varEqual
+    else:
+        # varEqual == False
+        varData = varData = [varInfo["varGene"], varInfo["varcDNAHGVS"], varType, varGenRef, varGenAlt, cDNARef, cDNAAlt]
+        return {"Consistent":varEqual,
```

Comment:
in general, variables should be lowercase (e.g. `consistent`). uppercase implies it's a class

Reply:
also looks like you don't need the `consistent` property at all, don't think it gets used later